### PR TITLE
[Excel] (Charts) Add chart data table article section

### PR DESCRIPTION
**Pull request by alison-mk**

### CODE REVIEW

File: docs/excel/excel-add-ins-charts.md [hunk 1]
```diff
@@ -183,6 +183,46 @@ Excel.run(function (context) {
...
+    var chartDataTableFormat = chartDataTable.format;
+    chartDataTableFormat.font.color = "#B76E79";
+    chartDataTableFormat.font.name = "Comic Sans";
+    chartDataTableFormat.border.color = "blue";
+
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+The following screenshot shows the data table that the preceding code sample creates, including the custom formatting of the data table.
```
Comment (AlexJerabek) [inline docs/excel/excel-add-ins-charts.md:222]: The clause about formatting the data table seems redundant.
```suggestion
The following screenshot shows the data table that the preceding code sample creates.
```

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
The section looks good. My only real concern for re-review is the about the image.

File: docs/excel/excel-add-ins-charts.md [hunk 2]
```diff
@@ -183,6 +183,46 @@ Excel.run(function (context) {
...
+
+![A chart with a data table, showcasing custom formatting of the data table.](../images/excel-charts-data-table.png)
```
[the remark this answers is not in the document]
Comment (alison-mk) [reply docs/excel/excel-add-ins-charts.md:224]: Good catch! Will fix.

File: docs/excel/excel-add-ins-charts.md [hunk 3]
```diff
@@ -183,6 +183,46 @@ Excel.run(function (context) {
 
 ![Chart with linear trendline in Excel.](../images/excel-charts-trendline-linear.png)
 
+## Add and format a chart data table
+
+You can access the data table element of a chart with the [`Chart.getDataTableOrNullObject`](/javascript/api/excel/excel.chart#getDataTableOrNullObject__) method. This method returns the [`ChartDataTable`](/javascript/api/excel/excel.chartdatatable) object. The `ChartDataTable` object has boolean formatting properties such as `visible`, `showLegendKey`, and `showHorizontalBorder`.
```
[the remark this answers is not in the document]
Comment (alison-mk) [reply docs/excel/excel-add-ins-charts.md:188]: I had the same thought and experience.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**